qutip/visualization.py

```diff
@@ -1718,17 +1699,17 @@ def plot_qubism(ket, theme='light', how='pairs',
     return fig, ax
 
 
-def plot_schmidt(ket, splitting=None,
-                 labels_iteration=(3, 2),
-                 theme='light',
-                 fig=None, ax=None, figsize=(6, 6)):
+def plot_schmidt(ket, splitting=None, labels_iteration=(3, 2),
+                 theme='light', *, fig=None, ax=None):
     """
     Plotting scheme related to Schmidt decomposition.
     Converts a state into a matrix (A_ij -> A_i^j),
     where rows are first particles and columns - last.
 
     See also: plot_qubism with how='before_after' for a similar plot.
 
+    NOTE: colorblind_safe does not apply because of its unique colormap
```

Comment:
This note format is recognized by the documentation builder and will look nice on the website.
```suggestion
    .. note:: 
    
        colorblind_safe does not apply because of its unique colormap
```